crates/ruff_dev/src/generate_docs.rs

```diff
@@ -37,9 +37,12 @@ pub(crate) fn main(args: &Args) -> Result<()> {
                 output.push('\n');
             }
 
-            let fix_kind = rule.fixable();
-            if matches!(fix_kind, FixKind::Always | FixKind::Sometimes) {
-                output.push_str(&fix_kind.to_string());
+            let fix_confidence = rule.fixable();
```

Comment:
Should probably update this variable name throughout

Reply:
is `FixAvailability` the name we want to go with? If so i'll change everything

Reply:
I like it.

Reply:
Works for me!